docs/Data loading, Migration/Migrating your existing Production DB.md

```diff
@@ -64,3 +64,105 @@ For assistance with a migration:
 
 - Existing customers can submit a [support case](https://mariadb.com/docs/skysql-previous-release/service-management/support/) to request assistance with a migration
 - New customers can [contact us](https://mariadb.com/docs/skysql-previous-release/contact/) to begin the migration planning process
+
+## Best Practices
+
+
+### Live Replication for Minimal Downtime
+
+To minimize downtime during migration, set up live replication from your source database to the SkySQL database. Follow these steps:
+
+1. **Obtain Binlog File and Position**: On the source database (MySQL or MariaDB), obtain the Binlog file name and its current position to track all database changes from a specific point in time.
+
+    ```sql
+    SHOW MASTER STATUS;
+    ```
+
+2. **Dump the Source Database**: Take a dump of your source database using `mysqldump` or `mariadb-dump`.
+
+    ```bash
+    mysqldump -u [username] -p --all-databases --master-data > dump.sql
```

Comment:
we need to call out skipping the mysql table and handling the user dump load separately as it will wipe out default skysql user causing monitor and access issues

Also we are omitting trigger and prodecudes, view, schedules